shell_integration/MacOSX/NextcloudIntegration/NextcloudIntegration.xcodeproj/project.pbxproj

```diff
@@ -710,7 +710,6 @@
 					"@executable_path/../Frameworks",
 					"@executable_path/../../../../Frameworks",
 				);
-				MACOSX_DEPLOYMENT_TARGET = 11.0;
```

Comment:
This breaks our Xcode integration

Reply:
Same goes for the other removals of the deployment targets in the pbxproj

Reply:
I don't think we can, but then again I don't think we need to, this doesn't affect the build we run via CMake (the deployment target on the xcodebuild run in the shell integration CMakeFiles is set by CMake). This is mainly for the IDE integration. I think we should set 11.0 as the deployment target for the File Provider extension and whatever our intended minimum target is for the Finder Sync Extension (10.13 or 10.14)